group-coordinator/src/main/java/org/apache/kafka/coordinator/group/streams/assignor/ProcessState.java

```diff
@@ -57,10 +57,6 @@ public int capacity() {
         return capacity;
     }
 
-    public int totalTaskCount() {
-        return assignedStandbyTasks().size() + assignedActiveTasks().size();
-    }
-
```

Comment:
nit: We don't need `totalTaskCount` anymore? I'm guessing no, just asking for clarification.